asf_heat_pump_suitability/config/base.yaml

```diff
@@ -1,3 +1,35 @@
+data:
+  geodata:
+    # OpenMap Local Geopackage
+    gb_os_openmap_local: "s3://asf-heat-pump-suitability/source_data/opmplc_gb.gpkg"
+    # OpenMap Local layers for certain grid squares
+    grid_square_os_openmap_local: "s3://asf-heat-pump-suitability/local_heat_planning/inputs/v042025_OSOpenMapLocal_geometries_selected/{square}/{square}_{layer}.shp"
+
+    boundaries:
+      # UK LA boundaries
+      UK_ons_lad_bounds: "s3://asf-heat-pump-suitability/source_data/Local_Authority_Districts_December_2023_Boundaries_UK_BFE_-2600600853110041429/LAD_DEC_2023_UK_BFE.shp"
```

Comment:
How are you differentiating `s3://asf-heat-pump-suitability/source_data/` from `s3://asf-heat-pump-suitability/local_heat_planning/inputs/`? i.e. what goes to which one?